### ISSUE TITLE
My Media Segregation Enhancement

### ISSUE THREAD

**Issue by Artiume** [jellyfin/jellyfin-web#782]
It was requested in chat to be able to provide multiple My Media libraries and dynamically select the libraries.

Picture Example

![image](https://user-images.githubusercontent.com/20715731/74096270-b0e70200-4aca-11ea-8f2e-2a642273d5f1.png)

**Comment by JustAMan** [comment]
Shouldn't this be tracked at https://features.jellyfin.org/ ?

**Comment by Artiume** [comment]
For major features, I'll put them there (I have about 5 on there now) but for minor end user enhancements, I'd rather keep them here where they're visible.

**Comment by JustAMan** [comment]
This one doesn't look like minor to me, it can require both backend and frontend work (and would probably require ATV client work at least as well).

**Comment by Artiume** [comment]
Yeah, probably. You can shift it to features if you want, I don't think it'll be done before EFcore

**Comment by thornbill** [comment]
Home screen enhancements are being tracked in Fider.
